tests/integration/test_syscollector/test_syscollector_configuration.py

```diff
@@ -0,0 +1,457 @@
+    evm.check_config(file_monitor=file_monitor, timeout=T_10)
+
+
+@pytest.mark.parametrize('configuration, metadata', zip(t5_configurations, t5_config_metadata), ids=t5_case_ids)
+def test_syscollector_scannig(configuration, metadata, set_wazuh_configuration,
+                              configure_local_internal_options_module, truncate_log_file,
+                              remove_agent_syscollector_info, daemons_handler_function):
+    '''
+    description: Check that the module sets the default values when the configuration block is empty.
+
+    test_phases:
+        - Configure syscollector.
+        - Configure modulesd in debug mode.
+        - Truncate the log.
+        - Restart analysisd, wazuh-db and modulesd
+
+    wazuh_min_version: 4.4.0
+
+    tier: 0
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Configuration loaded from the template located in `configuration` folder.
+        - metadata:
+            type: dict
+            brief: Test case metadata.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Set wazuh configuration using the configuration template.
+        - configure_local_internal_options_module:
+            type: fixture
+            brief: Configure the local internal options file.
+        - truncate_log_file:
+            type: fixture
+            brief: Truncate the log file before and after the test execution.
+        - remove_agent_syscollector_info:
+            type: fixture
+            brief: Removes the previous scan information.
+        - daemons_handler_function:
+            type: fixture
+            brief: Handler of Wazuh daemons for each test case.
+
+    assertions:
+        - Check if each scan is completed.
+        - Check if the synchronization is completed.
+
+    input_description:
+        - The `configuration_syscollector.yaml` file provides the module configuration for this test.
+        - The `case_test_scanning.yaml` file provides the test cases.
+    '''
+    file_monitor = FileMonitor(LOG_FILE_PATH) if sys.platform == 'win32' else None
+    # 60s + 2 seconds of margin because it includes the case when the agent starts for the first time
+    evm.check_has_started(file_monitor=file_monitor, timeout=T_60 + 2)
+    # Check general scan has started
+    evm.check_scan_started(file_monitor=file_monitor, timeout=T_10)
+
+    # Check that each scan was accomplished
+    scan_checks = [evm.check_hardware_scan_finished, evm.check_os_scan_finished, evm.check_network_scan_finished,
+                   evm.check_packages_scan_finished, evm.check_ports_scan_finished, evm.check_processes_scan_finished]
+    if sys.platform == 'win32':
+        scan_checks.append(evm.check_hotfixes_scan_finished)
+
+    for check in scan_checks:
+        # Run check
+        file_monitor = FileMonitor(LOG_FILE_PATH)
+        check(file_monitor=file_monitor, timeout=T_10)
```

Comment:
Improve name "check".

Reply:
my bad

Reply:
Done in [2fc9c3a](https://github.com/wazuh/wazuh-qa/pull/3584/commits/2fc9c3a925683eb7c02263f9c125cda163846187)